webclient/src/index.html

```diff
@@ -223,9 +223,9 @@
                   <router-link to="/about/${{_.footer.api.link}}$">${{_.footer.api.text}}$</router-link>
                 </div>
                 <div class="column col-auto links">
-                  <router-link to="/about/${{_.footer.contact.link}}$">${{_.footer.contact.text}}$</router-link><br>
+                  <a href="https://github.com/TUM-Dev/navigatum" class="c-hand">${{_.footer.sourcecode.text}}$</a><br>
                   <router-link to="/about/${{_.footer.privacy.link}}$">${{_.footer.privacy.text}}$</router-link><br>
-                  <router-link to="">${{_.footer.imprint.text}}$</router-link>
+                  <router-link to="/about/${{_.footer.imprint.link}}$">${{_.footer.imprint.text}}$</router-link>
```

Comment:
should we maybe include `/about/` into the link? I think this may confuse people.
If you dont think so we can keep it this way.

Reply:
I was slightly irritated by the diff view, but this is already there

Reply:
Jea.. noticed that your style is consisten across all links. Changing this would be a lot of work. I am closing this